src/main.rs

```diff
@@ -70,6 +67,77 @@ impl BFInterp {
             MetadataOperand => todo!(),
         }
     }
+    
+    fn load(&mut self, op: &Operand, dest: &llvm_ir::name::Name) {
+        match op {
+            LocalOperand { name, .. } => {
+                self.mem.insert(dest.clone(), *self.mem.get(name).unwrap());
+                self.mem.remove(name);
+            },
+            ConstantOperand(_) => todo!(),
+            MetadataOperand => todo!(),
+        }
+    }
+
+    fn call_func(&mut self, call: &Call) { 
+        let arg;    
+        match &call.arguments[1].0 {
+            LocalOperand { name, .. } => {
+                arg = self.mem.get(name).unwrap();
+            },
+            ConstantOperand(_) => todo!(),
+            MetadataOperand => todo!(),
+        }
+        
+        match &call.function {
+            Left(_) => todo!(),
+            Right(op) => {
+                match op {
+                    LocalOperand { .. } => todo!(),
+                    ConstantOperand(con_op) => {
+                        match con_op.as_ref() {
+                            Constant::GlobalReference { name, .. } => {
+                                match name {
+                                    Name(n) => {
+                                        if n.as_str() == "printf" {
```

Comment:
So there's an implicit assumption here that we print only the second argument, I think?

Should we put an assertion in case a different number of arguments are present?

Reply:
I think we should add some more prints to the test with different arguments, e.g.
```
printf("noargs");
printf("1 arg: %i", 1);
printf("2 args: %i %i", 1, 2);
printf("string arg: %s", "string");
```